lua/gitlab/actions/discussions/init.lua

```diff
@@ -442,6 +445,9 @@ M.rebuild_discussion_tree = function()
 end
 
 M.rebuild_unlinked_discussion_tree = function()
+  if M.unlinked_bufnr == nil then
+    return
+  end
```

Comment:
Someone can leave a note before initializing the discussion tree, we want to avoid running this function if the tree hasn't been built yet.